astro/cli/astro/cmd/fixtures/flags/merge_values.yaml

```diff
@@ -1,5 +1,8 @@
 ---
 
+terraform:
+  version: 0.11.7
```

Comment:
Yes it's required. It works for you because you have a `terraform` binary in your path. This ensures that the tests work in environments where Terraform is not already in the path.